### ISSUE TITLE
Dimensional Merchant

### ISSUE THREAD

**Issue by Forcecat01** [JoeAlisson/L2jOrg#478]
I always try to explain what the problem is, but I think the video will make everything clearer.

After a while, it is no longer possible to use the functions of the manager. only after re-entering the game for a short period of time you can press the buttons

https://youtu.be/a24CQxtDq5c

**Comment by vicochips** [comment]
What is this design of dimensional, thats not the good one for 272 protocol :O

**Comment by Forcecat01** [comment]
> What is this design of dimensional, thats not the good one for 272 protocol :O

Yes, it's not about the design, but about the fact that the function does not work through time. you have to re-enter the game to exchange tickets for charges

design can be made in any .u file and fixed in the system folder, suitable for any protocol

**Comment by Forcecat01** [comment]
I noticed such a moment. when the assistant stops working. you just talk to any NPC and the assistant works again for a while.
